AI_interface.py

```diff
@@ -24,28 +24,25 @@
 class DataWorker(object):
     def __init__(self, rank):
         self.agent_network = TFTNetwork()
-        self.prev_actions = [0 for _ in range(config.NUM_PLAYERS)]
         self.rank = rank
 
     # This is the main overarching gameplay method.
     # This is going to be implemented mostly in the game_round file under the AI side of things.
     def collect_gameplay_experience(self, env, buffers, global_buffer, storage, weights):
         self.agent_network.set_weights(weights)
-        agent = Batch_MCTSAgent(self.agent_network)
+        agent = MCTS(self.agent_network)
         while True:
             # Reset the environment
             player_observation = env.reset()
             # This is here to make the input (1, observation_size) for initial_inference
             player_observation = np.asarray(list(player_observation.values()))
             # Used to know when players die and which agent is currently acting
             terminated = {player_id: False for player_id in env.possible_agents}
-            # Current action to help with MuZero
-            self.prev_actions = [0 for _ in range(config.NUM_PLAYERS)]
 
             # While the game is still going on.
             while not all(terminated.values()):
                 # Ask our model for an action and policy
-                actions, policy = agent.batch_policy(player_observation, list(self.prev_actions))
+                actions, policy = agent.policy(player_observation)
```

Comment:
So, we are not using prev_action anymore? @silverlight6

Reply:
Nope, the previous actions are found as a by-product of the tree traversal algorithm in this current implementation, so no need to keep them.